wrapper/src/c_sharp_wrapper.cpp

```diff
@@ -56,7 +56,7 @@ bool CSharpWrapper::connect(
 	std::string msg;
 	try {
 		token = controller->init(msg, repo::lib::RepoConfig::fromFile(config));
-		return token;
+		return token != nullptr;
```

Comment:
Is this necessary? Would've thought it's casted anyway 😝

Reply:
I would have thought so too! But I've had this fail in AssetBundleCreator. Not sure why - perhaps it was specific to the compiler version I was using, but it definitely failed the cast (as in, it returned the wrong boolean value).